internal/csi-addons/networkfence/fencing.go

```diff
@@ -412,7 +413,37 @@ func (nf *NetworkFence) RemoveNetworkFence(ctx context.Context) error {
 		}
 		// remove ceph blocklist for each IP in the range mentioned by the CIDR
 		for _, host := range hosts {
-			err := nf.removeCephBlocklist(ctx, host, false)
+			err := nf.removeCephBlocklist(ctx, host, "0", false)
```

Comment:
@black-dragon74 Why is "0" used as nonce here ?

Reply:
This is to remove just the blacklist entry without specifying any extra details such as port and nonce. If you do not specify the port and nonce explicitly, ceph uses the default of 0/0 for port and nonce respectively.

Ex: `ceph osd blocklist rm x.x.x.x`, IP = x.x.x.x:**0/0**

Reply:
> This is to remove just the blacklist entry without specifying any extra details such as port and nonce. If you do not specify the port and nonce explicitly, ceph uses the default of 0/0 for port and nonce respectively.
> 
> Ex: `ceph osd blocklist rm x.x.x.x`, IP = x.x.x.x:**0/0**

Can you add it as comment just above this line ? 
its a bit confusing why "0" pops as nonce all of a sudden.